docs/interactive/hiding.md

```diff
@@ -104,7 +104,9 @@ See {ref}`content/toggle-admonitions` for more information on admonition dropdow
 ## Hide code cell content
 
 You can hide most cell elements of a page. The sections below describe how
-to hide each.
+to hide each using cell tags in MyST markdown.
+See [these instructions](https://jupyterbook.org/advanced/advanced.html#how-should-i-add-cell-tags-to-my-notebooks)
```

Comment:
this looks good! a couple quick thoughts:

1. Could you make the link text a bit more descriptive? Like "this guide on how to tag notebooks"?
2. Could you make the link itself an internal "sphinx" link, rather than hard-coding the HTML? (see how the other links in the book are written). E.g. you could add a `label` to that section with `(this-syntax)=`. And then link to it with `[My link text](this-syntax)`.

Reply:
Done, and I tested that it worked after building locally.